lib/Index/IndexSymbol.cpp

```diff
@@ -134,10 +134,18 @@ static SymbolKind getVarSymbolKind(const VarDecl *VD) {
   return SymbolKind::Variable;
 }
 
+SymbolInfo index::getSymbolInfoForModule(ModuleEntity Mod) {
+  SymbolInfo info{ SymbolKind::Unknown, SymbolSubKind::None,
```

Comment:
Should we give different parameters to clang modules?

Reply:
We can use the language kind to differentiate `SymbolLanguage::Swift` for Swift modules and generic `SymbolLanguage::C` for clang modules.